app/src/components/MealCard/MealCard.jsx

```diff
@@ -0,0 +1,230 @@
+import styles from "./MealCard.module.css";
+import { IoMdPeople } from "react-icons/io";
+import StyledButton from "../StyledButton/StyledButton";
+import { useState, useEffect } from "react";
+import TextField from "@mui/material/TextField";
+import axios from "axios";
+import StarRating from "../StarRating/StarRating.jsx";
+
+export default function MealCard({
+  title,
+  description,
+  price,
+  location,
+  max_reservation,
+  img_url,
+  id,
+}) {
+  const fetchReservationData = () => {
+    axios
+      .get(`http://localhost:3001/api/reservations/meal/${id}`)
```

Comment:
I suggest you use something like:
`${process.env.NEXT_PUBLIC_API_URL}`
instead of 
`http://localhost:3001/api/`

if the url changes you only have to change it one place ;)